sale_order_secondary_unit/tests/test_product_template.py

```diff
@@ -11,10 +11,19 @@ def setUpClass(cls):
         cls.product_uom_unit = cls.env.ref("uom.product_uom_unit")
         cls.product_attribute_model = cls.env["product.attribute"]
         cls.product_attribute_value_model = cls.env["product.attribute.value"]
-        # Create product attributes
-        cls.color_attribute = cls.product_attribute_model.create(
-            {"name": "Color", "create_variant": "always"}
```

Comment:
I think another possible solution is to define a unique attribute name, `Test color (sale_order_secondary_unit)` for example.

Reply:
Done